content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+in the upper right hand corner. A confirmation window will appear confirming the Language, click
+:guilabel:`Confirm`. Another confirmation will populate stating that the template will be submitted
+to Meta for review and approval. The :guilabel:`Status` will remain in :guilabel:`In review` until a
+decision has been made by Meta. Once the user receives an email confirming that the template has be
+approved the templates will need to be synced from the Odoo database.
+
+.. _whatsapp/sync:
+
+Syncing templates
+-----------------
+
+Templates must be synced on the Odoo database once they are approved by the Meta team. To do so,
+navigate to :menuselection:`WhatsApp app --> Configuration --> WhatsApp Business Accounts`. Select
+the configuration that should be synced. Under the section marked :menuselection:`Sending messages`,
+towards the bottom, click on :guilabel:`Sync Templates`. Meta will update the templates that are
+approved so that they can be utilized with various apps in the database.
+
+.. image:: whatsapp/sync-template.png
+   :align: center
+   :alt: Syncing Meta WhatsApp templates to the Odoo database, with the 'Sync Templates'
+         highlighted.
+
+A successful message in green should populate in the upper right hand corner with the number of
+templates updated.
+
+Using variables
+---------------
+
+Dynamic variables can be input on the WhatsApp template on the Meta business manager. This process
+should be completed during the initial layout and design through the Meta business console. Dynamic
+variables reference certain fields within the Odoo database to produce unique data in the WhatsApp
+message through the template. Dynamic variables are encoded to display fields from within the
+database referencing fields from within a model.
+
+.. example::
+   Many companies like to customize their WhatsApp messages with a personalized piece of customer
+   information to grab attention. This can be accomplished in Odoo by referencing a field within a
+   model by settings a dynamic variable. For example, a customer's name can be referenced in the
+   email from the :guilabel:`Customer` -->  field on the :guilabel:`Sales Order` model.
+
+First navigate to  `<https://business.facebook.com/wa/manage/home>`_. Then go to
+:menuselection:`Account tools --> Message templates`. Click on one and then :guilabel:`Edit
+template` or create a new template.
+
+Insert a variable by clicking on the :guilabel:`Add variable` button in the template. Variables can
+be added to the :guilabel:`Header`, :guilabel:`Body`, and :guilabel:`Footer` of the WhatsApp
+template.
+
+.. image:: whatsapp/add-variable.png
+   :align: center
+   :alt: Adding a variable in the body of a WhatsApp template.
+
+.. seealso::
+   :ref:`whatsapp/templates`
+
+After the template has been submitted and approved by Meta, the template needs to be synced with
+Odoo. To sync the template see this documentation: :ref:`whatsapp/sync`.
```

Comment:
If this is referencing the sync section above, I'd have an anchor to the sync section, then have the words 'sync the template with Odoo' as the link to that section instead of having a doc link here.